### PR TITLE
Remove backfill

### PR DESCRIPTION
**Pull request by Kludex**

### CODE REVIEW

**Comment by alexmojaki** [comment]
The file and fallback exporters can also be removed.

**Comment by alexmojaki** [comment]
(they don't have to be removed in this PR, but it's an option to resolve the missing coverage)

**Comment by Kludex** [comment]
> (they don't have to be removed in this PR, but it's an option to resolve the missing coverage)

I'm on it.

File: logfire/_internal/config.py [hunk 1]
```diff
@@ -60,7 +60,6 @@
 from .auth import DEFAULT_FILE, DefaultFile, is_logged_in
 from .config_params import ParamManager, PydanticPluginRecordValues
 from .constants import (
-    DEFAULT_FALLBACK_FILE_NAME,
```
Comment (alexmojaki) [inline logfire/_internal/config.py:63]: pls delete this value

File: logfire/_internal/config.py [hunk 2]
```diff
@@ -75,8 +74,6 @@
     ShowParentsConsoleSpanExporter,
     SimpleConsoleSpanExporter,
 )
-from .exporters.fallback import FallbackSpanExporter
```
Comment (alexmojaki) [inline logfire/_internal/config.py:78]: will you delete this?

**Review by alexmojaki — APPROVED** [review]

**Pull request merged**